config/ingest_journal_example.yml

```diff
@@ -11,7 +16,7 @@ tasks:
     do:
       - if: do_ocr
         do:
-          - convert.pdf_to_tif
+          - convert_pdf_to_tiff
```

Comment:
Hä, jetzt sind die Punkte wieder weg?

Reply:
Hä, aber wie können die tests denn dann durchlaufen?

Ich tippe auf einen Anzeigefehler in github, in der Commit-History siehts ja richtig aus ...